src/Control/Carrier/FossaApiClient/Internal/Core.hs

```diff
@@ -164,6 +164,6 @@ uploadArchive ::
   ) =>
   SignedURL ->
   FilePath ->
-  m LbsResponse
+  m ByteString
```

Comment:
I like this change.  It really should have been updated not to leak the network libraries.